experimental/streaming/exporter/spandata/spandata.go

```diff
@@ -57,12 +57,22 @@ func (s *spanReader) Read(data reader.Event) {
 		}
 	}
 
+	switch data.Type {
+	case observer.UPDATE_METRIC:
+		s.updateMetric(data)
+		return
+	}
+
 	span.Events = append(span.Events, data)
 
-	if data.Type == reader.FINISH_SPAN {
+	if data.Type == observer.FINISH_SPAN {
 		for _, r := range s.readers {
 			r.Read(span)
 		}
 		delete(s.spans, data.SpanContext)
 	}
 }
+
+func (s *spanReader) updateMetric(data reader.Event) {
+	// TODO aggregate
```

Comment:
It would be great to have this addressed before the PR is merged. Maybe this will uncover some problems with the API?

Reply:
Right. I was planning to attach some other metrics libraries instead. 
We're also looking to port over the OpenCensus metrics exporter, which will satisfy this TODO.

Reply:
I have the start of an SDK based on several-changes-ago of this branch, which I'll update today, and then we'll have a better look at the big picture.